lib/pp.rb

```diff
@@ -416,6 +416,26 @@ def pretty_print_cycle(q) # :nodoc:
   end
 end
 
+class Data # :nodoc:
+  def pretty_print(q) # :nodoc:
+    q.group(1, sprintf("#<data %s", PP.mcall(self, Kernel, :class).name), '>') {
```

Comment:
Why is `data` lower case?

Reply:
This is because `Struct#pretty_print` is a `struct`.
https://github.com/ruby/ruby/blob/30ab5cf835a64ade084a44d23ca2e12249b44608/lib/pp.rb#L401